.github/scripts/run_consensus.sh

```diff
@@ -2,21 +2,30 @@
 
 set -e
 
```

Comment:
It looks good for me. Would be also a good idea to make this script little bit more verbose, e.g. describe in a comment (or output to terminal?) that ids are generated from `//id` now and also what is the `sudo` account (use it somewhere here like in the previous version?) instead of making someone to read the whole script and looking for a default config value for `sudo`. In general, it would nice to see these components little bit more coupled together, i.e. instead of hard-coding and changing from time to time here and there a method by which those ids are derived, just use some configuration method (env variable, input arguments, whatever else).

Reply:
Good point, done